core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -31,6 +31,14 @@ function getThreadDumpEnabled() {
   return threadDumpEnabled;
 }
 
+function formatLossReason(removeReason, type, row) {
```

Comment:
Eh, uhoh. Seems like the JavaScript linter is broken by this:

```
added 118 packages in 1.482s

/__w/spark/spark/core/src/main/resources/org/apache/spark/ui/static/executorspage.js
   34:41  error  'type' is defined but never used. Allowed unused args must match /^_ignored_.*/u  no-unused-vars
   34:47  error  'row' is defined but never used. Allowed unused args must match /^_ignored_.*/u   no-unused-vars
   35:1   error  Expected indentation of 2 spaces but found 4                                      indent
   36:1   error  Expected indentation of 4 spaces but found 7                                      indent
   37:1   error  Expected indentation of 2 spaces but found 4                                      indent
   38:1   error  Expected indentation of 4 spaces but found 7                                      indent
   39:1   error  Expected indentation of 2 spaces but found 4                                      indent
  556:1   error  Expected indentation of 14 spaces but found 16                                    indent
  557:1   error  Expected indentation of 14 spaces but found 16                                    indent
```

Mind taking a look please?

Reply:
I just made a quick followup to fix up the build: https://github.com/apache/spark/pull/32541